### PR TITLE
Adding example-robot-data

### PR DESCRIPTION
**Pull request by jcarpent**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jcarpent** [comment]
@conda-forge/help-python-c This package is ready for review. Thanks in advance.

File: recipes/example-robot-data/bld.bat [hunk 1]
```diff
@@ -0,0 +1,18 @@
+mkdir build
```
Comment (scopatz) [inline recipes/example-robot-data/bld.bat:1]: This should be followed by a `if errorlevel 1 exit 1`

File: recipes/example-robot-data/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DPYTHON_EXECUTABLE=$PYTHON
+
+make install
```
Comment (scopatz) [inline recipes/example-robot-data/build.sh:12]: Probably should add `-j {CPU_COUNT}`

Comment (jcarpent) [reply recipes/example-robot-data/build.sh:12]: There is no compilation, so this is not useful.

**Comment by jcarpent** [comment]
@scopatz Done, I've handled your comments. Thanks a lot for your quick feedback.

**Comment by synapticarbors** [comment]
This recipe is still hanging out in `recipes` and it looks like the feedstock was never created.

**Comment by jcarpent** [comment]
@synapticarbors what are the next steps?

**Pull request merged**